test/groovy/com/sap/piper/ConfigurationHelperTest.groovy

```diff
@@ -29,6 +32,24 @@ class ConfigurationHelperTest {
         Assert.assertFalse(configuration.isPropertyDefined('something'))
     }
 
+    @Test
+    void testGetPropertyNestedLeafNodeIsString() {
+        def configuration = new ConfigurationHelper([a:[b: 'c']])
+        assertThat(configuration.getConfigProperty('a/b'), is('c'))
+    }
+
+    @Test
+    void testGetPropertyNestedLeafNodeIsMap() {
+        def configuration = new ConfigurationHelper([a:[b: [c: 'd']]])
+        assertThat(configuration.getConfigProperty('a/b'), is([c: 'd']))
+    }
+
+    @Test
+    void testGetPropertyNestedPathNotFound() {
+        def configuration = new ConfigurationHelper([a:[b: 'c']])
+        assertThat(configuration.getConfigProperty('a/c'), is((nullValue())))
+    }
+
```

Comment:
Tests for the edge cases added. The underlying tokenizer handles ```//k/``` like ```k```. Any "empty" delimiters (also leading and trailing) are silently ignored. Now we can start an academical fight for raising exceptions in case of miss-located delimiters ...